### PR TITLE
[MO] - Strimzi test container 0.102.0

### PR DESCRIPTION
**Pull request by see-quick**
This PR bumps up a new version of the Strimzi test container (0.102.0).

Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### CODE REVIEW

**Comment by see-quick** [comment]
When the 0.102.0 test container is released I will remove the additional repository (i.e., iostrimzi-1172).

**Review by scholzj — COMMENTED** [review]
Same as the other PR ... Should you you also update the Kafka version it uses in the tests? Or is it simply using always the latest version?

**Comment by see-quick** [comment]
> Same as the other PR ... Should you you also update the Kafka version it uses in the tests? Or is it simply using always the latest version?

It uses the latest version :)

**Review by Frawless — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -72,8 +72,8 @@
                         <url>https://oss.sonatype.org/content/repositories/snapshots</url>
                 </snapshotRepository>
                 <repository>
-                        <id>ossrh</id>
-                        <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+                    <id>ossrh</id>
+                    <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
```
[the remark this answers is not in the document]
Comment (see-quick) [reply pom.xml]: No, we don't. I have changed it.

**Review by scholzj — APPROVED** [review]

**Pull request merged**